vhdbuilder/packer/windows/windows-vhd-builder-sig.json

```diff
@@ -125,7 +125,8 @@
             "elevated_password": "{{.WinRMPassword}}",
             "environment_vars": [
                 "ProvisioningPhase=3",
-                "WindowsSKU={{user `windows_sku`}}"
+                "WindowsSKU={{user `windows_sku`}}",
+                "SKIP_EXTENSION_CHECK={{ user `SKIP_EXTENSION_CHECK` }}"
```

Comment:
It would be nice to avoid all the stupid variables having to be explicitly being passed through - if we used a well-known env var prefix maybe we could automate it in the pipeline and packager?

Reply:
It is a good idea!